### PR TITLE
Incorporate semiprecious mod

### PR DESCRIPTION
**Pull request by theraven262**
- [x] Retexturing
- [x] Clean up the code to match the rest of the repository
- [x] Tweak oregen params (when we get to mapgen/game balance)

### CODE REVIEW

**Comment by theraven262** [comment]
#26

File: mods/semiprecious/depends.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+default
```
Comment (lisacvuk) [inline mods/semiprecious/depends.txt]: depends.txt is deprecated now - we should switch this to mod.conf

**Comment by theraven262** [comment]
Further balancing oregen and crafting can be done after the merge.

**Comment by wrynvera** [comment]
Opps, responded to the wrong thread lol
The looks good is for this.

**Pull request merged**